### DISCUSSION TITLE
How to Stop Motion Sensor from receiving data?

### DISCUSSION THREAD

**Discussion by chiqors** [ionic-team/capacitor#5245] [Q&A] [answered]
How can i stop the motion sensor listener function? also how can i set the cooldown for receiving motion sensor data (interval)?

**Comment by jcesarmobile** [comment, marked as answer]
I’ve updated the example on the docs with how to remove the listeners.

https://capacitorjs.com/docs/apis/motion#permissions

**Reply by chiqors** [reply]
is there anyway to set interval for cooldown for motion data to be received?

**Reply by jcesarmobile** [reply]
No, you can’t control the interval. You can check the interval on the returned data and discard some values if needed.

**Reply by chiqors** [reply]
i see, well, that's okay. Thanks for the answer!
